zetaclient/core_context/zeta_core_context.go

```diff
@@ -179,3 +179,15 @@ func (c *ZetaCoreContext) Update(
 		c.currentTssPubkey = tssPubKey
 	}
 }
+
```

Comment:
Why not just provide the observer/client object in the method as context and chainParams can be inferred from it?

Reply:
I think the context is a low level concept and used by other packages. The context itself only need to know basic data structures, e.g. `ChainParams` to make its life easy.